core/worker.h

```diff
@@ -99,8 +97,8 @@ class Worker {
  private:
   volatile worker_status_t status_;
 
-  int wid_;  /* always [0, MAX_WORKERS - 1] */
-  int core_; /* TODO: should be cpuset_t */
+  int wid_;   // always [0, kMaxWorkers - 1]
+  int core_;  // TODO: should be cpuset_t
```

Comment:
While updating this, want to make the change to cpuset_t as the TODO indicates?  (If it's more involved, never mind.)